plugin/storage/cassandra/spanstore/dbmodel/unique_tags.go

```diff
@@ -16,27 +16,26 @@ package dbmodel
 
 import "github.com/uber/jaeger/model"
 
-// GetAllUniqueTags creates a list of all unique tags found in a span and process
-func GetAllUniqueTags(span *model.Span) []TagInsertion {
-	process := span.Process
-	allTags := span.Tags
-	allTags = append(allTags, process.Tags...)
+// GetAllUniqueTags creates a list of all unique tags from a set of filtered tags.
+func GetAllUniqueTags(span *model.Span, tagFilter TagFilter) []TagInsertion {
+	tags := tagFilter.FilterProcessTags(span.Process.Tags)
```

Comment:
- allTags was a better name imo (and would cause fewer spurious changes in this diff)
- are you sure you don't need to init to nil? if FilterProcessTags simply returns its arg, and the underlying array had extra capacity, then append might just add to that, creating potential side effects.

Reply:
Just wrote some primitive code: https://play.golang.org/p/5MbPA7Siya looks like the existing process tags doesn't get touched even if it has extra capacity

Reply:
https://play.golang.org/p/jI1r1-T9jY